internal/core/ngt/option.go

```diff
@@ -37,7 +37,7 @@ var (
 		WithDimension(0),
 		WithDefaultRadius(-1.0),
 		WithDefaultEpsilon(0.01),
-		WithDefaultPoolSize(1),
+		WithDefaultPoolSize(10000),
```

Comment:
🤔 
```suggestion
		WithPoolSize(10000),
```

Reply:
I think it's okay to naming defaultPoolSize, If Createindex API request does not include pool size we use defaultPoolSize

Reply:
as kpango said, it is named "Defaultxx" because the pool size should be set by CreateIndex API request payload and it is used if the payload doesn't have a specific pool size.